doc/source/serve/advanced-guides/dev-workflow.md

```diff
@@ -77,11 +77,23 @@ After you're done testing, you can shut down Ray Serve by interrupting the `serv
 (ServeController pid=9865) INFO 2022-08-11 11:47:19,929 controller 9865 deployment_state.py:1257 - Removing 1 replicas from deployment 'HelloDeployment'.
 ```
 
-Note that rerunning `serve run` will redeploy all deployments. To prevent redeploying those deployments whose code hasn't changed, you can use `serve deploy`; see the [Production Guide](serve-in-production) for details.
+Note that rerunning `serve run` redeploys all deployments. To prevent redeploying the deployments whose code hasn't changed, you can use `serve deploy`; see the [Production Guide](serve-in-production) for details.
+
+### Local Testing Mode
```

Comment:
can you please add a markdown `NOTE` (see other pages for example syntax) that indicates this feature is experimental